<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>979</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0,1">
    <item row="1" column="0">
     <widget class="QPushButton" name="startPushButton">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>384</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="inputGroupBox">
      <property name="title">
       <string>Input</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="5" column="5">
        <widget class="QDoubleSpinBox" name="wzDoubleSpinBox">
         <property name="decimals">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item row="6" column="4">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>γ</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Θ</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QDoubleSpinBox" name="tAlDoubleSpinBox">
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="singleStep">
          <double>0.001000000000000</double>
         </property>
         <property name="value">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>ωx</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QDoubleSpinBox" name="wxDoubleSpinBox">
         <property name="decimals">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QDoubleSpinBox" name="psiDoubleSpinBox">
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="minimum">
          <double>-99.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>V</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="VDoubleSpinBox">
         <property name="value">
          <double>25.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>ba</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QDoubleSpinBox" name="baDoubleSpinBox">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>0.280000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>df</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>da</string>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>dr</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>fi</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="dfDoubleSpinBox">
         <property name="minimum">
          <double>-99.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QDoubleSpinBox" name="daDoubleSpinBox">
         <property name="minimum">
          <double>-20.000000000000000</double>
         </property>
         <property name="maximum">
          <double>20.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="5">
        <widget class="QDoubleSpinBox" name="fiDoubleSpinBox">
         <property name="minimum">
          <double>-35.000000000000000</double>
         </property>
         <property name="maximum">
          <double>-12.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="5">
        <widget class="QDoubleSpinBox" name="drDoubleSpinBox">
         <property name="minimum">
          <double>-25.000000000000000</double>
         </property>
         <property name="maximum">
          <double>25.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QDoubleSpinBox" name="LDoubleSpinBox">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>0.853400000000000</double>
         </property>
        </widget>
       </item>
       <item row="7" column="3">
        <widget class="QDoubleSpinBox" name="tAtDoubleSpinBox">
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="singleStep">
          <double>0.002000000000000</double>
         </property>
         <property name="value">
          <double>0.001000000000000</double>
         </property>
        </widget>
       </item>
       <item row="6" column="3">
        <widget class="QDoubleSpinBox" name="thetaDoubleSpinBox">
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="value">
          <double>20.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Jz</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QDoubleSpinBox" name="SDoubleSpinBox">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>0.214300000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>ρ</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>ωy</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Ψ</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="4">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>ωz</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="5">
        <widget class="QDoubleSpinBox" name="gammaDoubleSpinBox"/>
       </item>
       <item row="0" column="5">
        <widget class="QDoubleSpinBox" name="JzDoubleSpinBox">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>0.389300000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="JxDoubleSpinBox">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>0.066700000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QDoubleSpinBox" name="wyDoubleSpinBox">
         <property name="decimals">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Jy</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Jx</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QDoubleSpinBox" name="JyDoubleSpinBox">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>0.452100000000000</double>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>tc</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>L</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>S</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>ta</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="rhoDoubleSpinBox">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>1.225000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="6">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="7" column="4">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>te</string>
         </property>
        </widget>
       </item>
       <item row="7" column="5">
        <widget class="QDoubleSpinBox" name="endTimeSpinBox">
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>1.000000000000000</double>
         </property>
         <property name="value">
          <double>10.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Plot</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="5" column="0">
        <widget class="QCheckBox" name="plotFiCheckBox">
         <property name="text">
          <string>fi</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QCheckBox" name="plotServo5CheckBox">
         <property name="text">
          <string>servo5</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="plotServo1CheckBox">
         <property name="text">
          <string>servo1</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="plotServo2CheckBox">
         <property name="text">
          <string>servo2</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QCheckBox" name="plotServo3CheckBox">
         <property name="text">
          <string>servo3</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="plotWxCheckBox">
         <property name="text">
          <string>ωx</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="plotWyCheckBox">
         <property name="text">
          <string>ωy</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QCheckBox" name="plotMzCheckBox">
         <property name="text">
          <string>mz</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="plotMxCheckBox">
         <property name="text">
          <string>mx</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="plotMyCheckBox">
         <property name="text">
          <string>my</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QCheckBox" name="plotGammaCheckBox">
         <property name="text">
          <string>γ</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QCheckBox" name="plotWzCheckBox">
         <property name="text">
          <string>ωz</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="plotThetaCheckBox">
         <property name="text">
          <string>Θ</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="plotPsiCheckBox">
         <property name="text">
          <string>Ψ</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="plotServo4CheckBox">
         <property name="text">
          <string>servo4</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QCheckBox" name="plotDfCheckBox">
         <property name="text">
          <string>df</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QCheckBox" name="plotDaCheckBox">
         <property name="text">
          <string>da</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QCheckBox" name="plotDrCheckBox">
         <property name="text">
          <string>dr</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2" rowspan="4">
     <widget class="QCustomPlot" name="plotWidget" native="true"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
